DOCKER.md

```diff
@@ -49,19 +46,12 @@ Alternatively, you can run the container manually:
 # Create a directory for bundles
 mkdir -p ./bundles
 
-# Run the container with the default entrypoint
-docker run -it --rm \
-  -v "$(pwd)/bundles:/data/bundles" \
-  -e SBCTL_TOKEN="your_token_here" \
-  -e MCP_BUNDLE_STORAGE="/data/bundles" \
-  mcp-server-troubleshoot:latest
-
-# Run the container with the MCP server stdio mode for use with MCP clients
+# Run the container
 docker run -i --rm \
   -v "$(pwd)/bundles:/data/bundles" \
   -e SBCTL_TOKEN="your_token_here" \
   -e MCP_BUNDLE_STORAGE="/data/bundles" \
```

Comment:
This should actually be a default so it doesn't need to be added here. Remove it.